plugwise/constants.py

```diff
@@ -437,7 +437,7 @@
     # Added measurements from actuator_functionalities/thermostat_functionality
     "lower_bound": {ATTR_UNIT_OF_MEASUREMENT: TEMP_CELSIUS},
     "upper_bound": {ATTR_UNIT_OF_MEASUREMENT: TEMP_CELSIUS},
-    "resolution": {ATTR_UNIT_OF_MEASUREMENT: None},
+    "resolution": {ATTR_UNIT_OF_MEASUREMENT: TEMP_KELVIN},
```

Comment:
Why Kelvin here? :)

Reply:
As in more range probably, but I read resolution as precision not neccesarily a unit of measurement?

Reply:
I was always taught that 25 degr. C - 24 degr. C = 1 degr. K.
1 degr. C can also be interpreted as absolute 1 degr. C. 
To avoid this confusion, I was taught to use degr. K as the unit for temperature difference.

But, yeah, resolution is a different thing, I will change to degr. C
The resolution is for the lower- and upper-bound temperatures so it has to have the same unit of measurement.